frontend/src/main/pages/PlayPage.js

```diff
@@ -90,9 +89,8 @@ export default function PlayPage() {
     mutationbuy.mutate(userCommons)
   };
 
-
   const onSuccessSell = () => {
-    toast(`Cow sold!`);
+    //toast(`Cow sold!`);
```

Comment:
Can we remove this commented out line of code?

Reply:
I just removed both "cow bought" and "cow sold" comments!